### DISCUSSION TITLE
[Feature] SVG Sprite support

### DISCUSSION THREAD

**Discussion by formula1** [ikemen-engine/Ikemen-GO#889] [Feature Request]
I know it's absurd and the feature may not get used (I don't plan to use it) but svgs look really great.

Here's a fun example
https://www.svgator.com/

### Advantages
- they look great
- I don't know personally but they may be easier to animate than creating individual pixel frames. I'm thinking flash games

As mentioned by @slzero-X

- They can be scaled up and down beautifully
- The animations have the potential to be incredibly smooth. Possibly supporting absurd fps rates

### Long Term
- maybe there's a possibility that the hitboxes can be based on the SVG shapes. And allow the use to toggle a piece to be a hit, hurt or uninteractable

I'm lucky #420, that's cute

**Comment by lokpro** [comment]
reminds me of (Adobe) Flash games.

**Comment by formula1** [comment]
@lokpro I was thinking that also. While I like dablimg in pixel art it might be easier for artists to animate svgs then having to create individual pixel frames. It may sound far fetched but maybe the hotboxes can be based on the SVG as well.

**Comment by ghost** [comment]
This could mean we could make sprites that are very high in quality and lose very little when scaled down. This also allows us to use 2 dimensional skeletons for very smooth animations that can be played back at any speed. Imagine the possibilities this will give us.

**Comment by formula1** [comment]
@slzero-X I mean, I'm no artist but I agree. The possibilities seem amazing
